metrics/character/README.md

```diff
@@ -0,0 +1,94 @@
+---
+title: CharacTER
+emoji: 🔤
+colorFrom: orange
+colorTo: red
+sdk: gradio
+sdk_version: 3.0.2
+app_file: app.py
+pinned: false
+tags:
+- evaluate
+- metric
+- machine-translation
+description: >-
+  CharacTer is a novel character level metric inspired by the commonly applied translation edit rate (TER).
+---
+
+# Metric Card for CharacTER
+
+## Metric Description
+CharacTer is a novel character level metric inspired by the commonly applied translation edit rate (TER). It is 
+defined as the minimum number of character edits required to adjust a hypothesis, until it completely matches the
+reference, normalized by the length of the hypothesis sentence. CharacTer calculates the character level edit
+distance while performing the shift edit on word level. Unlike the strict matching criterion in TER, a hypothesis
+word is considered to match a reference word and could be shifted, if the edit distance between them is below a
+threshold value. The Levenshtein distance between the reference and the shifted hypothesis sequence is computed on the
+character level. In addition, the lengths of hypothesis sequences instead of reference sequences are used for
+normalizing the edit distance, which effectively counters the issue that shorter translations normally achieve lower
+TER.
+
+## Intended Uses
+CharacTER was developed for machine translation evaluation.
+
+## How to Use
+
+```python
+import evaluate
+character = evaluate.load("character")
+preds = ["this week the saudis denied information published in the new york times",
+         "this is in fact an estimate"]
+refs = ["saudi arabia denied this week information published in the american new york times",
+        "this is actually an estimate"]
+results = character.compute(references=refs, predictions=preds)
+
+```
+### Inputs
+- **predictions**: a single prediction or a list of predictions to score. Each prediction should be a string with
+     tokens separated by spaces.
+- **references**: a single reference or a list of reference for each prediction. Each reference should be a string with
+     tokens separated by spaces.
```

Comment:
Small thing: I think this currently fails on single examples since the CER package tries to calculate stdev, but that requires at least two data points (see below for example). Do you know if there's a way to pass a flag down to `cer` to turn this off for single examples?

```
p = ["this week the saudis denied information published in the new york times"]
r = ["saudi arabia denied this week information published in the american new york times"]
results = character.compute(references=r, predictions=p)
```

<img width="1028" alt="Screen Shot 2022-09-13 at 11 56 21 AM" src="https://user-images.githubusercontent.com/31600291/189986779-31b99a4f-f55d-48c0-b7e3-39f1e3ae60e6.png">

Passing in strings directly also triggers a mismatch error (`Mismatch in the number of predictions (71) and references (82)`).

Reply:
Ah, good call. I'll update `cer` one of these days to make sure that does not give any issues.